Travis building becomes to fail

Investigate and fix that issue.

#### Solution:
```
(django-app) marcosranes@ubuntu21-vbox:~/Desktop/django-app$ git branch 13
(django-app) marcosranes@ubuntu21-vbox:~/Desktop/django-app$ git checkout 13
Switched to branch '13'
(django-app) marcosranes@ubuntu21-vbox:~/Desktop/django-app$ exit
exit
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ pipenv run pytest .
================================================== test session starts ==================================================
platform linux -- Python 3.9.7, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /home/marcosranes/Desktop/django-app
plugins: cov-3.0.0
collected 1 item                                                                                                        

django-app/tests/test_example.py .                                                                                [100%]

=================================================== 1 passed in 0.02s ===================================================
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ pipenv run flake8 .
./django-app/tests/test_example.py:4:1: W391 blank line at end of file
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ nano django-app/tests/test_example.py 
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ cat django-app/tests/test_example.py 
def test_int():
    assert 1 == 1
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ pipenv run flake8 .
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ git status
On branch 13
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   django-app/tests/test_example.py

no changes added to commit (use "git add" and/or "git commit -a")
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ git add .
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ git commit -m "fixed Python code removed blank lines
> close #13"
[13 eb2f614] fixed Python code removed blank lines close #13
 1 file changed, 2 deletions(-)
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ git push origin 13
Enumerating objects: 9, done.
Counting objects: 100% (9/9), done.
Delta compression using up to 4 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (5/5), 479 bytes | 479.00 KiB/s, done.
Total 5 (delta 1), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
remote: 
remote: Create a pull request for '13' on GitHub by visiting:
remote:      https://github.com/PortalNetZone/django-app/pull/new/13
remote: 
To github.com:PortalNetZone/django-app.git
 * [new branch]      13 -> 13
marcosranes@ubuntu21-vbox:~/Desktop/django-app$ 
```

--- a/django-app/tests/test_example.py
+++ b/django-app/tests/test_example.py
@@ -1,4 +1,2 @@
 def test_int():
     assert 1 == 1
-
-

